### DISCUSSION TITLE
Pinecone Upsert API getting     "message": "Error: vectorsService.upsertVector - Error: error: Tenant or user not found",

### DISCUSSION THREAD

**Discussion by marsalan06** [FlowiseAI/Flowise#3636] [General]
`
curl --location 'http://0.0.0.0:3000/api/v1/vector/upsert/e3b5fe1c-db87-45aa-a029-9bac8cb33470' \
--form 'files=@"/home/arsalan/Desktop/arsalan/rag/groq_rag/data/cricket.pdf"' \
--form 'metadata="{\"source\":\"class_XII_bio\"}"' \
--form 'pineconeNamespace="class_XII"'
`

This is my upsert api , i am getting error on flowise as 

{
    "statusCode": 500,
    "success": false,
    "message": "Error: vectorsService.upsertVector - Error: error: Tenant or user not found",
    "stack": {}
}

flowise-1  | 2024-12-04 19:23:18 [INFO]: ⬆️ POST /api/v1/vector/upsert/e3b5fe1c-db87-45aa-a029-9bac8cb33470
flowise-1  | 2024-12-04 19:23:19 [ERROR]: error: Tenant or user not found
flowise-1  | Error: error: Tenant or user not found
flowise-1  |     at Pinecone_VectorStores.upsert (/usr/local/lib/node_modules/flowise/node_modules/flowise-components/dist/nodes/vectorstores/Pinecone/Pinecone.js:80:27)
flowise-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
flowise-1  |     at async buildFlow (/usr/local/lib/node_modules/flowise/dist/utils/index.js:420:37)
flowise-1  |     at async upsertVector (/usr/local/lib/node_modules/flowise/dist/utils/upsertVector.js:167:32)
flowise-1  |     at async Object.upsertVectorMiddleware (/usr/local/lib/node_modules/flowise/dist/services/vectors/index.js:9:16)
flowise-1  |     at async upsertVectorMiddleware (/usr/local/lib/node_modules/flowise/dist/controllers/vectors/index.js:18:29)
flowise-1  | 2024-12-04 19:23:19 [ERROR]: [server]: Error: Error: error: Tenant or user not found
flowise-1  | Error: Error: error: Tenant or user not found
flowise-1  |     at buildFlow (/usr/local/lib/node_modules/flowise/dist/utils/index.js:496:19)
flowise-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
flowise-1  |     at async upsertVector (/usr/local/lib/node_modules/flowise/dist/utils/upsertVector.js:167:32)
flowise-1  |     at async Object.upsertVectorMiddleware (/usr/local/lib/node_modules/flowise/dist/services/vectors/index.js:9:16)
flowise-1  |     at async upsertVectorMiddleware (/usr/local/lib/node_modules/flowise/dist/controllers/vectors/index.js:18:29)

**Comment by Max-Falcon** [comment]
had the same issue as you. If you are using a postgres record manager in your flow, go to supabase and check your postgres db. Most likely it is paused and that is what derives the issue.

**Reply by saatchi-david** [reply]
Just had this exact issue. Thanks @Max-Falcon for the share!
